Docker_Container/dockerfile

```diff
@@ -0,0 +1,21 @@
+# pull official base image
+FROM node:13.12.0-alpine
+
+# set working directory
+WORKDIR /app
+
+# add `/app/node_modules/.bin` to $PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+# install app dependencies
+COPY package.json ./
+COPY yarn.json ./
+COPY package-lock.json ./
+RUN npm install --silent
```

Comment:
Remember if `yarn` is being used as the package manager you'd have to run `yarn install` instead of `npm install` or the `package-lock.json` file will be generated instead